apps/book/serializers.py

```diff
@@ -1,11 +1,26 @@
 from rest_framework import serializers
-# from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy as _
 
-from apps.book.models import Book
+from apps.book.models import Book, WishList
 
 
 class BookSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Book
         fields = '__all__'
+
+
+class WishListSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = WishList
+        fields = ('book', )
+
+    def validate(self, attrs):
+        request = self.context.get('request')
+        attrs['user'] = request.user
```

Comment:
Let's create a base class for auto set created_by and modified_by if they exists in the model.